doozer/doozerlib/build_info.py

```diff
@@ -166,24 +191,148 @@ def get_installed_package_build_dicts(self) -> Dict[str, Dict]:
 
         return self._cache[cn]
 
-    def get_archive_pullspec(self):
+    def get_pullspec(self):
         """
-        :return: Returns an internal pullspec for a specific archive image.
+        :return: Returns an internal pullspec for a specific image.
                  e.g. 'registry-proxy.engineering.redhat.com/rh-osbs/openshift-ose-openshift-controller-manager:rhaos-4.6-rhel-8-containers-candidate-53809-20210722091236-x86_64'
         """
         return self._archive['extra']['docker']['repositories'][0]
 
-    def get_archive_digest(self):
+    def get_digest(self):
         """
         :return Returns the archive image's sha digest (e.g. 'sha256:1f3ebef02669eca018dbfd2c5a65575a21e4920ebe6a5328029a5000127aaa4b')
         """
         digest = self._archive["extra"]['docker']['digests']['application/vnd.docker.distribution.manifest.v2+json']
-        if not digest.startswith("sha256:"):  # It should start with sha256: for now. Let's raise an error if this changes.
+        # It should start with sha256: for now. Let's raise an error if this changes.
+        if not digest.startswith("sha256:"):
             raise ValueError(f"Received unrecognized digest {digest} for archive {self.get_archive_id()}")
         return digest
 
+    def get_manifest_list_digest(self):
+        return self.build_record_inspector.get_manifest_list_digest()
+
+
+class KonfluxImageInspector(ImageInspector):
```

Comment:
Unsure if I like it that two implementations of an abstract class live in the same file. We have similar in elliott's bzutil, and I am always confused while navigating/grepping that. Maybe an idea to split implementations in their own files for human sake? 

Obviously, this is not a blocker.

Reply:
Think it's a good idea, I will keep this as a follow up to limit changes we're doing in this PR (and so that I don't have to run test builds again for now).